### PR TITLE
bug(tiering): add prints to find crash flow

### PR DESCRIPTION
**Pull request by adiholden**

### CODE REVIEW

**Comment by theyueli** [comment]
> @theyueli lets try to run this branch, before you run memtier the second time we will set the vmodule flag to see the flow of the crash

sounds good

File: src/server/tiered_storage.cc [hunk 1]
```diff
@@ -480,6 +481,7 @@ bool TieredStorage::PrepareForOffload(DbIndex db_index, PrimeIterator it) {
 }
 
 void TieredStorage::CancelOffload(DbIndex db_index, PrimeIterator it) {
+  VLOG(2) << "PrepareForOffload:" << it->first.ToString();
```
Comment (theyueli) [inline src/server/tiered_storage.cc:484]: typo: PrepareForCancelOffLoad

**Review by theyueli — DISMISSED** [review]

**Comment by kostasrim** [comment]
@adiholden @theyueli is something pending or shall we close this ?

**Comment by adiholden** [comment]
@kostasrim I will edit the unit tests for merge later. keep this open

**Pull request closed without merging**